### PR TITLE
Allow for alpha values to be received by onColorChange props when the…

### LINKED ISSUE #5766: ColorPicker should return alpha value in onColorChanged callback
**Describe the feature that you would like added**
The ColorPicker should return an alpha value whenever the color or alpha changes.  It's currently impossible to tell what the alpha value is set to.
**What component or utility would this be added to**
ColorPicker
**Have you discussed this feature with our team, and if so, who**
No
**Additional context/screenshots**

### PR DESCRIPTION
**Pull request by bishwenduk029**
…re is change in color. The alpha value is optional

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #5766
- [x] Include a change request file using `$ npm run change`

#### Description of changes

When a color change event happens in the ColorPicker component, now alpha value along with the color sting is also extracted and passed to the onColorChange props.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
Thanks for your contribution and fixing that issue @bishwenduk029! 

It looks like there's a small lint error in the unit test that is failing Travis on this PR:

```shell
[office-ui-fabric-react 04:30:48] Pass: jest (129.191s)
/home/travis/build/OfficeDev/office-ui-fabric-react/packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx:44:9
ERROR: 44:9  prefer-const  Identifier 'a' is never reassigned; use 'const' instead of 'let'.
```

Once fixed, this should be good to merge with a sign-off.

**Comment by bishwenduk029** [comment]
@KevinTCoughlin , thanks for acknowledging means a lot. @phkuo , sure I will make the change to allow for the same, update the unit test cases ASAP and verify lint error for another push ASAP.

**Comment by cliffkoh** [comment]
+1 on what Kevin said. Thanks for fixing this "help wanted" issue :)

**Comment by bishwenduk029** [comment]
Thanks you so much @cliffkoh 👍 :)

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { mount } from 'enzyme';
 import { ColorPicker } from './ColorPicker';
 import { ColorPickerBase, IColorPickerState } from './ColorPicker.base';
 import { IColorPicker, IColorPickerProps } from './ColorPicker.types';
+import { IColor } from 'office-ui-fabric-react/lib/utilities/color/colors';
```
[the remark this answers is not in the document]
Comment (bishwenduk029) [reply packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: Yes @cliffkoh , it has I change and confirm

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { mount } from 'enzyme';
...
+import { getColorFromString } from '../../utilities/color/colors';
```
[the remark this answers is not in the document]
Comment (cliffkoh) [reply packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: One important thing is to make sure you use the capital `U` version (the Utilites.ts file, not the utilities folder)

**Comment by cliffkoh** [comment]
@phkuo @KevinTCoughlin can you please review/test the change? :) (I have hopefully fixed the build break introduced in f0aa0a3)

**Comment by bishwenduk029** [comment]
Thanks a ton @KevinTCoughlin & @cliffkoh for all you help support.....it was awesome :)

**Pull request merged**